### ISSUE TITLE
Factor Weighted Cumulative Return plot for each day/period

### ISSUE THREAD

**Issue by luca-s** [quantopian/alphalens#65]
Factor Weighted Cumulative Return plot

Only the 1 day forward returns graph is plotted in  'Factor Weighted Long/Short Portfolio Cumulative Return'. I expected to see plots for each day/period  forward return ('days' argument passed to create_factor_tear_sheet) .
I might have a factor that shows alpha only after X or Y days and I might not be interested to see 1 day forward returns plots, so I would pass days = (X, Y) to create_factor_tear_sheet and I would expect to see the plots for X and Y forward returns. At least it would be nice to have this feature as an option if not the default behaviour.

**Comment by jameschristopher** [comment]
https://github.com/quantopian/alphalens/pull/83
